packages/jest-cli/src/TestScheduler.js

```diff
@@ -173,6 +174,7 @@ export default class TestScheduler {
         // $FlowFixMe
         testRunners[config.runner] = new (require(config.runner): TestRunner)(
           this._globalConfig,
+          this._options.changedFiles,
```

Comment:
can you put this in an object? `type RunData = { changedFiles: ?Set<Path> }` or something like that

Reply:
maybe put `changedFiles` into `TestSchedulerContext` instead, and pass that whole thing?